packages/react-performance/src/test/performance-effect-server.test.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
```

Comment:
this test will probably fail eventually, if JSDom ever adds any API for `performance`. It's best to set the `jest-env` here to ensure you get a clean, `performance`-free place to run the test. Separately, not a huge fan of the name of the test not matching the name of the file it tests.